packages/common/src/lib/entity/entity-table/entity-table.component.ts

```diff
@@ -461,7 +482,26 @@ export class EntityTableComponent implements OnInit, OnChanges, OnDestroy {
       if (all[0]) {
         this.enableEdit(all[0]);
       }
-      this.dataSource.data = all;
+      this.processedRowData = [];
+      all.map((record) => {
+        const rowData: RowData = {
+          record,
+          cellData: undefined
+        };
+        const cellData: CellData = {};
+        this.template.columns.map((column) => {
+          const value = this.getValue(record, column);
+          cellData[column.name] = {
+            class: this.getCellClass(record, column),
+            value,
+            isUrl: this.isUrl(value),
+            isImg: this.isImg(value)
+          };
+        });
+        rowData.cellData = cellData;
+        this.processedRowData.push(rowData);
+      });
+      this.dataSource.data = this.processedRowData;
```

Comment:
Si dataSource.data est la même chose que processedRowData, on pourrait simplifier:
```suggestion
      this.dataSource.data = all.map((record) => {
        return {
          record,
          cellData: this.template.columns.reduce((cellData, column) => {
            const value = this.getValue(record, column);
            cellData[column.name] = {
              class: this.getCellClass(record, column),
              value,
              isUrl: this.isUrl(value),
              isImg: this.isImg(value)
            };
            return cellData;
          }, {})
        };
      });
```